packages/fxa-auth-server/lib/senders/renderer/index.ts

```diff
@@ -139,6 +116,24 @@ class Renderer extends Localizer {
     };
   }
 
+  private async getGlobalTemplateValues(
+    context: TemplateContext
+  ): Promise<GlobalTemplateValues> {
+    // We must use 'require' here, 'import' causes an 'unknown file extension .ts'
+    // error. Might be a config option to make it work?
+    try {
```

Comment:
I put this here because I figured in that follow up issue, I'd be testing this function anyway and want to make sure the error is handled and I thought it might make the stack trace slightly easier to see it was coming from this function, BUT tbh I think it's because I'm just used to wrapping async calls that feel error-prone in a try/catch. Maybe we don't need it.